core/src/main/scala/org/apache/spark/network/netty/NettyBlockTransferService.scala

```diff
@@ -56,7 +56,7 @@ private[spark] class NettyBlockTransferService(
     _port: Int,
     numCores: Int,
     driverEndPointRef: RpcEndpointRef = null)
-  extends BlockTransferService {
+  extends BlockTransferService with Logging {
 
```

Comment:
"with Logging" is more consistent and natural here.

Reply:
Given the `logger` exposed via `BlockStoreClient`, why use `Logging` here ?